src/components/calculator.js

```diff
@@ -1,23 +1,72 @@
 /*eslint-disable*/
+import { calculateNewValue } from '@testing-library/user-event/dist/utils';
 import React from 'react';
+import calculate from '../logic/calcualte'
```

Comment:
- [ ] Kindly fix the spelling in the importing of the calculate file as l in logic has to capital it should be like this `import calculate from '../Logic/calcualte'`.

Reply:
@tingamapuro04 great work implementing the changes. kindly check your spelling its suppose to be a capital `L` for `Logic` not `logic`.